packages/rspack-cli/src/rspack-cli.ts

```diff
@@ -18,7 +18,7 @@ import { loadRspackConfig } from "./utils/loadConfig";
 import { Mode } from "@rspack/core/src/config";
 import { RspackPluginInstance, RspackPluginFunction } from "@rspack/core";
 
-type RspackEnv = "development" | "production";
+type Command = "serve" | "buid";
```

Comment:
buid => build

Reply:
why doesn't our spell checker detect this?

Reply:
https://github.com/speedy-js/rspack/blob/1778fe9a9149ff328c8f649aa472ac29d91cb49a/.typos.toml#L7 it seems because of this line